kaldb/src/main/java/com/slack/kaldb/logstore/search/SearchQuery.java

```diff
@@ -46,10 +47,10 @@ public String toString() {
         + endTimeEpochMs
         + ", howMany="
         + howMany
-        + ", bucketCount="
-        + bucketCount
         + ", chunkIds="
-        + String.join(",", chunkIds)
+        + chunkIds
```

Comment:
is the default behaviour of List<String> == `String.join(",", chunkIds)` ?

Reply:
Good catch - I just used IntelliJ to generate this method and didn't catch the difference. It is indeed similar to what you listed, but with extra brackets:

```java
List<String> stringList = List.of("1", "2", "3");
System.out.println(stringList);
```

```bash
[1, 2, 3]
```